integration_tests/commands/async/json_test.go

```diff
@@ -1378,3 +1379,62 @@ func TestJsonARRTRIM(t *testing.T) {
 		})
 	}
 }
+
+func TestJsonSTRAPPEND(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
+	simpleJSON := `{"name":"John","age":30}`
+
+	testCases := []struct {
+		name     string
+		setCmd   string
+		getCmd   string
+		expected interface{}
+	}{
+		{
+			name:     "STRAPPEND to nested string",
+			setCmd:   `JSON.SET doc $ ` + simpleJSON,
+			getCmd:   `JSON.STRAPPEND doc $.name " Doe"`,
+			expected: []interface{}{int64(8)},
+		},
+		{
+			name:     "STRAPPEND to multiple paths",
+			setCmd:   `JSON.SET doc $ ` + simpleJSON,
+			getCmd:   `JSON.STRAPPEND doc $..name "baz"`,
+			expected: []interface{}{int64(7)},
+		},
+		{
+			name:     "STRAPPEND to non-string",
+			setCmd:   `JSON.SET doc $ ` + simpleJSON,
+			getCmd:   `JSON.STRAPPEND doc $.age " years"`,
+			expected: []interface{}{"(nil)"},
+		},
+		{
+			name:     "STRAPPEND with empty string",
+			setCmd:   `JSON.SET doc $ ` + simpleJSON,
+			getCmd:   `JSON.STRAPPEND doc $.name ""`,
+			expected: []interface{}{int64(4)},
+		},
+		{
+			name:     "STRAPPEND to non-existent path",
+			setCmd:   `JSON.SET doc $ ` + simpleJSON,
+			getCmd:   `JSON.STRAPPEND doc $.nonexistent " test"`,
+			expected: []interface{}{},
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			FireCommand(conn, "DEL doc")
+			defer FireCommand(conn, "DEL doc")
```

Comment:
Why're two `DEL` invocations required here? Can we have it as part of logic above as @apoorvyadav1111 suggested: https://github.com/DiceDB/dice/pull/882/files#r1782010024

Reply:
Made the changes for this using FLUSHDB to delete all the keys before each tests